.github/workflows/release.yaml

```
@@ -44,6 +44,14 @@ jobs:
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v1
 
+      # Setup Cosign
+      - name: Install Cosign
+        uses: sigstore/cosign-installer@v2.1.0
```

```suggestion
        uses: sigstore/cosign-installer@v2.3.0
```
Latest version of the GH action